### ISSUE TITLE
Spotify tiling full-screen action spams i3 with configure requests

### ISSUE THREAD

**Issue by i3bot** [i3/i3#1124]
[Originally reported by tony@…]
(When my Spotify client is in tiling mode, trying to full-screen the window with the command `fullscreen` fails. Instead, the window remains non-full-screen and spams i3 with configure requests.

```
11/22/2013 02:00:44 PM - handlers.c:handle_configure_request:297 - window 0x01e0001a wants to be at 1600x0 with 1278x1024
11/22/2013 02:00:44 PM - handlers.c:handle_configure_request:328 - Configure request!
11/22/2013 02:00:44 PM - con.c:con_is_floating:412 - checking if con 0x1203890 is floating
11/22/2013 02:00:44 PM - xcb.c:fake_absolute_configure_notify:104 - fake rect = (1600, 6, 638, 1018)
11/22/2013 02:00:44 PM - handlers.c:handle_configure_request:297 - window 0x01e0001a wants to be at 1600x0 with 1278x1024
11/22/2013 02:00:44 PM - handlers.c:handle_configure_request:328 - Configure request!
11/22/2013 02:00:44 PM - con.c:con_is_floating:412 - checking if con 0x1203890 is floating
11/22/2013 02:00:44 PM - xcb.c:fake_absolute_configure_notify:104 - fake rect = (1600, 6, 638, 1018)
```

It will stop with these configure requests when I issue another `fullscreen` command (but of course, will not go into full screen mode).

If the container is floating, the `fullscreen` command works as expected.

Spotify client is built on the qt framework. I couldn't reproduce the issue with kvirc, but there is the possibility it may affect other qt clients.

**Comment by Airblader** [comment]
@acrisci I'm assuming you authored this issue? It's fairly old now; can it still be reproduced?

**Comment by acrisci** [comment]
@Airblader yes, spotify cannot be fullscreened from the tiling state for me.

**Comment by Airblader** [comment]
I just installed spotify and it immediately starts spamming with configure requests, even without fullscreen.

**Comment by simonnagl** [comment]
@acrisci: Just a suggestion. This does not resolve this issue. Do you know about https://play.spotify.com/? maby it fits to your needs. The only disadvantage I see: It uses flash.

**Comment by Airblader** [comment]
I've tried looking into this more. A detail @acrisci mentioned, but wasn't necessarily clear: fullscreening it when it is floated works just fine.

Sometimes. Other times it seems to instantly leave fullscreen again. Looking at the log I could see that spotify sends a client message asking us to leave fullscreen. It'd be really helpful to study spotify's code, but it doesn't seem to be open source. Which also means we don't support this.

Anyway, I've installed spotify beta to see if the issue persists – and it doesn't. It's working just fine there.
